pkg/loadbalancers/loadbalancers_test.go

```
@@ -195,15 +198,20 @@ func verifyHTTPSForwardingRuleAndProxyLinks(t *testing.T, j *testJig) {
 	t.Helper()
 	lbName := j.namer.LoadBalancer(ingressName)
 
-	um, err := j.fakeGCE.GetURLMap(j.UMName(lbName))
-	tps, err := j.fakeGCE.GetTargetHTTPSProxy(j.TPName(lbName, true))
+	key, _ := composite.CreateKey(j.fakeGCE, j.UMName(lbName), defaultScope)
+	um, err := composite.GetUrlMap(j.fakeGCE, key, defaultVersion)
```

ignored the err? or handle it

---

Done